core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -127,7 +132,7 @@
   MASTER_HEARTBEAT_INTERVAL_MS(Name.MASTER_HEARTBEAT_INTERVAL_MS, 1000),
   MASTER_HOSTNAME(Name.MASTER_HOSTNAME, null),
   MASTER_JOURNAL_FLUSH_BATCH_TIME_MS(Name.MASTER_JOURNAL_FLUSH_BATCH_TIME_MS, 5),
-  MASTER_JOURNAL_FOLDER(Name.MASTER_JOURNAL_FOLDER, "${alluxio.home}/journal/"),
+  MASTER_JOURNAL_FOLDER(Name.MASTER_JOURNAL_FOLDER, "${alluxio.work.directory}/journal"),
```

Comment:
dropped the trailing slash for consistency. I audited all usages of this property and double-checked that we are always agnostic to whether it contains a trailing slash or not